### PR TITLE
Fix AST migration steps for created model directives

### LINKED ISSUE #205: Index directives not working in preview-16.2
Datamodel:

``` prisma
model User {
  id         Int       @id @unique
}

model Post {
  id        Int       @id @unique
}

model Comment {
  id        Int       @id @unique
}

model Like {
  id        Int       @id @unique
  user      User
  post      Post
  comment   Comment

  @@index([user])
  @@index([post])
  @@index([comment])
}
```

Error:

``` json
{
  "jsonrpc": "2.0",
  "error": {
    "code": 4466,
    "message": "An error happened. Check the data field for details.",
    "data": {
      "type": "DataModelErrors",
      "code": 1001,
      "errors": [
        "Argument \"\" is already specified.",
        "Argument \"fields\" is missing in attribute \"@index\".",
        "Argument \"\" is already specified.",
        "Argument \"\" is already specified.",
        "Argument \"fields\" is missing in attribute \"@index\".",
        "Argument \"fields\" is missing in attribute \"@index\"."
      ]
    }
  },
  "id": 2
}
```

### PR DESCRIPTION
**Pull request by tomhoule**
Fixes #205

### CODE REVIEW

**Comment by pimeys** [comment]
https://github.com/prisma/prisma-engine/issues/205

File: migration-engine/core/src/migration/datamodel_differ/models.rs [hunk 1]
```diff
@@ -127,13 +127,13 @@ fn fields_match(previous: &ast::Field, next: &ast::Field) -> bool {
...
-fn is_regular(directive: &&ast::Directive) -> bool {
+pub(super) fn is_regular(directive: &&ast::Directive) -> bool {
     !is_repeated(directive)
 }
 
 /// See ModelDiffer docs.
-fn is_repeated(directive: &&ast::Directive) -> bool {
-    REPEATED_MODEL_DIRECTIVES.contains(&directive.name.name.as_str())
+pub(super) fn is_repeated(directive: &&ast::Directive) -> bool {
+    REPEATED_MODEL_DIRECTIVES.contains(&dbg!(directive.name.name.as_str()))
```
Comment (pimeys) [inline migration-engine/core/src/migration/datamodel_differ/models.rs]: `dbg!`

File: migration-engine/connectors/sql-migration-connector/src/lib.rs [hunk 2]
```diff
@@ -15,16 +15,7 @@ pub use error::*;
 pub use sql_migration::*;
 
 use migration_connector::*;
-<<<<<<< HEAD
```
Comment (pimeys) [inline migration-engine/connectors/sql-migration-connector/src/lib.rs]: ...

**Comment by pimeys** [comment]
Works and indices are there also.

File: migration-engine/connectors/sql-migration-connector/src/lib.rs (hunk 2 above)
Comment (tomhoule) [reply migration-engine/connectors/sql-migration-connector/src/lib.rs]: been doing a lot of rebasing of the async branch to avoid big conflicts

**Pull request merged**